packages/web/src/utils/utils.ts

```diff
@@ -108,3 +108,23 @@ export const emitEvent = (
 
   return true;
 };
+
+// Log validation error for required properties in components
```

Comment:
Just a small suggestion to use clarify the params -- the most important thing I'd like to see is the one for the `name`

```suggestion
/* Log validation error for required properties in components
 * @param name - name of the component i.e. <gcds-*>
 * @param errorArr - array of attributes with errors
 * @param optionalAttrsArrToRemove - array of optional attributes with errors to be removed from this error message
 */
```